### ISSUE TITLE
[Feature Req] Enable scroll wheel ignore option on clock module

### ISSUE THREAD

**Issue by jackloomen** [Alexays/Waybar#1529]
Such that scroll wheel will change workspaces as it would on empty bar space.
This feature is nice to have when the the clock is placed center of the bar.

**Comment by LukashonakV** [comment]
Hi @jackloomen ,
This https://github.com/Alexays/Waybar/pull/2019 implements ability to redefine module action on your wish (any mouse buttons, wills). Once this is merged you can redefine Clock actions which were defined by default on scroll. SO in such case you won't get into actions conflict between workspaces and clock module

**Comment by LukashonakV** [comment]
Hi @jackloomen  #2019 is merge into the master.
So now your are able to redefine Clock actions on your wish. 
Please see: 
1. [Module actions config](https://github.com/Alexays/Waybar/wiki/Configuration#module-actions-config)
2. [Wiki clock](https://github.com/Alexays/Waybar/wiki/Module:-Clock)

If it's Ok for you , please close an issue.
Thank you

**Comment by jackloomen** [comment]
Thanks.
